.gitlab/corona/pipeline.yml

```diff
@@ -190,6 +201,9 @@ release allocation:
     # problems if it's not set.
     LBANN_CI_RUN_WEEKLY: ${LBANN_CI_RUN_WEEKLY}
 
+    # Preserve any spack-build-* directories to allow for faster testing
+    GIT_CLEAN_FLAGS: -ffdx -e spack-build-*
+
```

Comment:
I don't understand why these are needed? The catch tests run with the build, the relevant logs are stored as artifacts, and the `ci_tests/` directory doesn't use the build tree at all.

Reply:
Oh nevermind. This is to accelerate successive pipelines in which one happens to have gotten the same runner (and Spack hashes LBANN the same way). My concern, though, is that, as hashes accumulate, we will just eat up our disk space with stale build directories. We may want to augment this approach with a cron job to clean that waste up (e.g., directories with mtime older than a week or something).

Reply:
@benson31 I agree that we should have a cron job to clean these directories and spack environments.